lib/room.cpp

```diff
@@ -52,18 +52,25 @@
 #include "converters.h"
 #include "syncdata.h"
 
+#include "e2ee.h"
+
 #include <QtCore/QHash>
 #include <QtCore/QStringBuilder> // for efficient string concats (operator%)
 #include <QtCore/QPointer>
 #include <QtCore/QDir>
 #include <QtCore/QTemporaryFile>
 #include <QtCore/QRegularExpression>
 #include <QtCore/QMimeDatabase>
+#include <QtCore/QJsonDocument>
 
 #include <array>
 #include <functional>
 #include <cmath>
 
+#include <session.h> // QtOlm
+#include <groupsession.h> // QtOlm
+#include <message.h> // QtOlm
```

Comment:
Please place these above the standard library includes. I'm yet to merge the clang-format branch that will enforce the order of `#include`s.